### PR TITLE
Add instructions to disable icon display

### PR DESCRIPTION
**Pull request by herrbischoff**
I had to dig quite a bit, including in the code itself to figure out how to do this. This shouldn't be needed for a a straight-forward configuration change. So, in order to save (new) users some time, I amended the documentation.

### CODE REVIEW

**Comment by gegoune** [comment]
I disagree with assumption that it is not straight forward to disable icons therefore don't see this change as warranted. It's all documented.

File: README.md [hunk 1]
```diff
@@ -81,6 +81,26 @@ For complete list of available configuration options see [:help nvim-tree.setup]
 
 Each option is documented in `:help nvim-tree.OPTION_NAME`. Nested options can be accessed by appending `.`, for example [:help nvim-tree.view.mappings](doc/nvim-tree-lua.txt)
 
+Using the icons requires a [patched font](https://www.nerdfonts.com/). If you
+don't have one installed, you're probably seeing undesirable "?" box glyphs.
+To disable the display of icons altogether, you can use the following minimal
+configuration:
```
Comment (alex-courtis) [inline README.md:87]: This is mostly covered in `:help nvim-tree-introduction` but clarity is nice.

We can just add "To disable the display of icons altogether set each of |renderer.icons.show| to false." in that section.

**Review by alex-courtis — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (herrbischoff) [reply README.md:87]: Good idea, thanks for the suggestion.

Comment (alex-courtis) [reply README.md:87]: Agreed

Comment (herrbischoff) [reply README.md:87]: Sounds good. I will make the changes shortly.

**Comment by alex-courtis** [comment]
Pushed https://github.com/kyazdani42/nvim-tree.lua/commit/08db5a576dbdc44e3996e643978c8e6a5a460d56

**Pull request closed without merging**